core/cmd/local_client.go

```diff
@@ -249,9 +258,23 @@ func (cli *Client) RebroadcastTransactions(c *clipkg.Context) (err error) {
 
 	err = store.Start()
 	if err != nil {
-		return err
+		return cli.errorOut(err)
+	}
+
+	if store.Config.EnableBulletproofTxManager() {
```

Comment:
Can we get a test around this new functionality? It'd be more of an integration test, but I'm specifically concerned that the CLI can complete everything while a chainlink node holds the advisory lock.